src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/transformer/TransformerTest.kt

```diff
@@ -19,8 +18,7 @@ class TransformerTest {
       id = UUID.randomUUID(),
       name = "A Course",
       type = "A type",
-      prerequisites = emptyList(),
-      audience = emptyList(),
+      prerequisites = emptySet(),
```

Comment:
The tests for audience have been removed in this commit `Make Prerequisite persistent`, which doesn't quite feel like the right place for it - should it be removed/updated elsewhere? Perhaps in `Incorporate the Course audience property by rebasing from main and adjust for JPA integration.`?